features/dish/lib/screen/dish_screen.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:core_ui/core_ui.dart';
+import 'package:domain/domain.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:navigation/navigation.dart';
+
+class DishScreen extends StatelessWidget {
+  final MenuItemEntity _model;
```

Comment:
Is it entity from data? if -yes : it is mistake. Here only domain models (or from state).